javascript/javascript1/week1/week1.js

```diff
@@ -23,13 +23,13 @@ else{
 
 //Housey pricey (A house price estimator)
 console.log("********** Housey pricey (A house price estimator) *********");
-let peterHouseDetails = [8,10,10,100, 2500000];
-let juliaHouseDetails = [5,11,8,70,1000000]
-let peterVolumeInMeters = peterHouseDetails[0]*peterHouseDetails[1]*peterHouseDetails[2];
-let juliaVolumeInMeters = juliaHouseDetails[0]*juliaHouseDetails[1]*juliaHouseDetails[2];
+const peterHouseDetails = [8,10,10,100, 2500000];
+const juliaHouseDetails = [5,11,8,70,1000000]
```

Comment:
I ran your code and I can see it works as expected - great work and nice you aren't afraid to make use of arrays and good you changed the variables to using const!

What I can recommend is getting into the habit of being as explicit as possible when it comes to giving your variables names.

The next time you review your code it will be hard to see at first glance what `[8,10,10,100,2500000]`. I would therefore recommend naming each of the variables so it's easier when you use it in the code.

I would recommend something along the lines of:
```
const peterHouseWidth = 8;
const peterHouseDepth = 10;
const peterHouseHeight = 10;
const peterHouseGardenSize = 100;
```

It will make it easier if your code base was to grow and you have to revisit old work.
`const peterVolumeInMeters = peterHouseWidth * peterHouseDepth * peterHouseHeight * peterHouseGardenSize`

Than having to remember what was in index 0, 1, 2 of the peterHouse array.

`const peterVolumeInMeters = peterHouseDetails[0]*peterHouseDetails[1]*peterHouseDetails[2]`